src/frontend/cypress/e2e/chat-list-tests/search-chats.cy.js

```diff
@@ -0,0 +1,75 @@
+describe("search spec", () => {
+  it("filters all", () => {
+    cy.login({ user: "user1", password: "password1" });
+
+    cy.getLitElement(
+      "il-app,il-chat,il-sidebar,il-conversation-list,il-search,il-input-ricerca"
```

Comment:
È meglio prendere la string e metterla in una costante, dato che viene usata più volte uguale nel file, dandole anche un nome descrittivo.
Si può fare anche per gli altri file di test.